### ISSUE TITLE
confignetwork does not seem to set the IP addresses of bond0 

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#2486]
Not sure if I'm executing these steps correctly here...   I set the following attributes for the node: 

```
chdef frame45cn10 nictypes.enP3p3s0f0=ethernet nictypes.enP3p3s0f1=ethernet \
   nictypes.bond0=bond nicdevices.bond0="enP3p3s0f0|enP3p3s0f1" 
chdef frame45cn10 nicips.bond0=40.1.45.10
```

Then I run the ``confignetwork`` postscript, that executes successfully and I do see the bonded interface created, but without an IP address 

```
[root@frame45cn10 network-scripts]# ifconfig
bond0: flags=5187<UP,BROADCAST,RUNNING,MASTER,MULTICAST>  mtu 1500
        inet 40.1.45.10  netmask 255.0.0.0  broadcast 40.255.255.255
        inet6 fe80::9abe:94ff:fe59:ea24  prefixlen 64  scopeid 0x20<link>
        ether 98:be:94:59:ea:24  txqueuelen 1000  (Ethernet)
        RX packets 197  bytes 66768 (65.2 KiB)
        RX errors 0  dropped 2  overruns 0  frame 0
        TX packets 668  bytes 84620 (82.6 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

enP3p3s0f0: flags=6211<UP,BROADCAST,RUNNING,SLAVE,MULTICAST>  mtu 1500
        ether 98:be:94:59:ea:24  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 5  bytes 444 (444.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
        device interrupt 251  memory 0x250000000000-2500007fffff

enP3p3s0f1: flags=6211<UP,BROADCAST,RUNNING,SLAVE,MULTICAST>  mtu 1500
        ether 98:be:94:59:ea:24  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
        device interrupt 252  memory 0x250001000000-2500017fffff
...
```

This is all that is created... 
```
[root@frame45cn10 network-scripts]# cat ifcfg-bond0
DEVICE="bond0"
BOOTPROTO="none"
NAME="bond0"
BONDING_MASTER="yes"
ONBOOT="yes"
USERCTL="no"
TYPE="Bond"
BONDING_OPTS="mode=802.3ad miimon=100"
DHCLIENTARGS="-timeout 200"
```

Once I add the following into the ifcfg-bond0, and reload the adapter, it shows up...

```
IPADDR=40.1.45.10
NETMASK=255.0.0.0
```

I thought this should happen automatically?

**Comment by bybai** [comment]
Hi Victor,

I think you do not add network for your node.
You should add nicnetworks.bond0, and add this networks in the network table.
```
chdef frame45cn10 nicnetworks.bond0=......
```

**Comment by whowutwut** [comment]
@bybai Thanks! I thought I was missing something.  I didn't realize that we needed to define all the additional networks that we want to configure.  Let me give that a try... I was following the documentation under here: http://xcat-docs.readthedocs.io/en/stable/guides/admin-guides/manage_clusters/ppc64le/diskless/customize_image/cfg_network.html 

Perhaps I will make some changes to the doc in a PR and split up the bridge config and bond configuration.

**Comment by bybai** [comment]
Hi @whowutwut , If we want to configure ip for network adapter, we should add nicnetworks for the network adapter. We should add this into doc.

**Comment by whowutwut** [comment]
@bybai yea I think @gurevichmark tried your suggestion today. I'll see if he can create a PR to change some of the wording in the docs

**Comment by gurevichmark** [comment]
Yes. I will create a pull request, once we figure out the correct set of steps for bonding

**Comment by gurevichmark** [comment]
Fixed by PR #2534

**Comment by whowutwut** [comment]
Closing with documentation change.
